packages/opal-client/opal_client/policy_store/options.py

```diff
@@ -0,0 +1,20 @@
+from enum import Enum
+
+from pydantic import BaseModel, Field
+
+
+class WaitStrategy(str, Enum):
+    fixed = "fixed"
```

Comment:
Maybe just add some docs referencing the tenacity details

Reply:
Are the new comments in 96d5f799930e92c67e2b248668a0ea374e42bdd5 enough?